gnosis/gnosis-spec.ini

```diff
@@ -296,7 +301,7 @@ callData: #abiCallData("encodeTransactionData", (
             #uint256(NONCE) ))
 callValue: 0
 activeAccounts: SetItem(MASTER_COPY_ID) SetItem(PROXY_ID)
-                SetItem(TO) SetItem(GAS_TOKEN) SetItem(REFUND_RECEIVER) _:Set
+                _:Set
```

Comment:
In this case, you can just drop the whole `activeAccounts` parameter here, as it is already defined in the `[root]` section.

Reply:
I mean, we need to specify the `ORIGIN_ID` in the activeAccounts cell. I don't think an inactive account can send a transaction, which you can double-check though.

Reply:
Done. Waiting the tests. Last time there was a spurious failure on Jenkins. Couldn't reproduce locally.